src/gui/tray/unifiedsearchresultslistmodel.cpp

```diff
@@ -324,26 +324,46 @@ bool UnifiedSearchResultsListModel::isSearchInProgress() const
     return !_searchJobConnections.isEmpty();
 }
 
-void UnifiedSearchResultsListModel::resultClicked(const QString &providerId, const QUrl &resourceUrl) const
+
+void UnifiedSearchResultsListModel::resultClicked(
+    const QString &providerId, const QUrl &resourceUrl, const QString &subline, const QString &title
+) const
```

Comment:
I know our code style might not be well defined but arguments should not start on a new line (as visible for other methods)
```suggestion
void UnifiedSearchResultsListModel::resultClicked(const QString &providerId,
                                                                                           const QUrl &resourceUrl,
                                                                                           const QString &subline,
                                                                                           const QString &title) const
```